shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -173,9 +173,17 @@ - (void)testkeyboardWillChangeFrameWillStartKeyboardAnimation {
                     @"UIKeyboardAnimationDurationUserInfoKey" : [NSNumber numberWithDouble:0.25],
                     @"UIKeyboardIsLocalUserInfoKey" : [NSNumber numberWithBool:isLocal]
                   }];
+
+  XCTestExpectation* expectation = [self expectationWithDescription:@"update viewport"];
```

Comment:
@gaaclarke would you mind reviewing this test change? I just added it.

Reply:
LGTM